specification/app/resource-manager/Microsoft.App/stable/2022-05-01/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -159,6 +176,23 @@
                 ]
               }
             ],
+            "initContainers": [
+              {
```

Comment:
Is there a mapping for initcontainer to a container?

Reply:
Hi @p-bouchon , want to ask is this comment resolved? I'm ok for others in the PR